lib/pages/main_page.dart

```diff
@@ -1,22 +1,128 @@
 import 'package:flutter/material.dart';
+import 'package:fokus/utils/theme_config.dart';
+import 'package:lottie/lottie.dart';
+
 import 'package:fokus/data/model/user/caregiver.dart';
 import 'package:fokus/utils/app_locales.dart';
+import 'package:fokus/utils/dialog_utils.dart';
 
 class MainPage extends StatefulWidget {
-	@override
-	_MainPageState createState() => new _MainPageState();
+  @override
+  _MainPageState createState() => new _MainPageState();
 }
 
 class _MainPageState extends State<MainPage> {
-	@override
-	Widget build(BuildContext context) {
-		var user = ModalRoute.of(context).settings.arguments as Caregiver;
-
-		return Scaffold(
-			appBar: AppBar(
-				title: Text(AppLocales.of(context).translate('page.caregiverPanel.header.title', {'name': user.name})),
+  @override
+  Widget build(BuildContext context) {
+    var user = ModalRoute.of(context).settings.arguments as Caregiver;
+    return Scaffold(
+      backgroundColor: ThemeConfig.mainBackgroundColor,
+      body: Column(
+        mainAxisAlignment: MainAxisAlignment.center,
+        crossAxisAlignment: CrossAxisAlignment.center,
+        children: <Widget>[
+          Center(
+            child: Lottie.asset('assets/animation/sunflower_with_title_rotate_only.json', width: 280)),
+          Center(
+            child: Container(
+							width: 240,
+							padding: EdgeInsets.all(4.0),
+							child: FlatButton(
+								onPressed: () => { Navigator.of(context).pushNamed('/caregiver-panel-page', arguments: user) },
+								color: ThemeConfig.caregiverButtonColor,
+								padding: EdgeInsets.all(20.0),
+								child: Row(
+									mainAxisAlignment: MainAxisAlignment.center,
+									children: <Widget>[
+										Text(
+											'${AppLocales.of(context).translate("page.main.introduction")} ',
+											style: TextStyle(
+												fontSize: 18,
+												fontWeight: FontWeight.normal,
+												color: ThemeConfig.lightTextColor
+											)
+										),
+										Text(
+											'${AppLocales.of(context).translate("page.main.caregiver")} ',
+											style: TextStyle(
+												fontSize: 18,
+												fontWeight: FontWeight.bold,
+												color: ThemeConfig.lightTextColor
+											)
+										),
+										Icon(
+											Icons.arrow_forward,
+											color: ThemeConfig.lightTextColor,
+											size: 26
+										)
```

Comment:
In such cases RichText widget is useful
```suggestion
										RichText(
											text: TextSpan(
												text: AppLocales.of(context).translate("page.main.introduction") + ' ',
												style: TextStyle(fontSize: 18, color: ThemeConfig.lightTextColor),
												children: <TextSpan>[
													TextSpan(
														text: AppLocales.of(context).translate("page.main.caregiver"),
														style: TextStyle(fontWeight: FontWeight.bold)
													),
												],
											),
										),
										Padding(
										  padding: const EdgeInsets.only(left: 4),
										  child: Icon(
										  	Icons.arrow_forward,
										  	color: ThemeConfig.lightTextColor,
										  	size: 26
										  ),
										)
```

Reply:
IMO RichText has a better use when there is a lot of text. It seems like the second text is depending on the first one because it's placed one level lower. It's harder to read too 😕

I added the padding as you suggested, looks better (first use of AppBoxProperites class - yay).

Reply:
Well, it depends style-wise (just changes the weight). But I agree those things can be subjective so not forcing anything :)

Reply:
Then if that style is not going to be commonly used (so to end up in global styles) lets define it once in build method and use four times using ```copyWith()``` when necessary.